libs/design/tree/src/interfaces/tree-mode.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
Can you rename this file to `tree-render-mode.ts` ?

Reply:
dd11deb72